bedwars-plugin/src/main/java/com/tomkeuper/bedwars/language/Bangla.java

```diff
@@ -274,6 +274,7 @@ public Bangla() {
         yml.addDefault(Messages.PLAYER_DIE_REWARD_GOLD, "%bw_lang_prefix%&6+%bw_amount% %bw_meaning%");
         yml.addDefault(Messages.ARENA_MAX_BUILD_LIMIT_REACHED, "&cMax build height limit reached!");
         yml.addDefault(Messages.ARENA_MIN_BUILD_LIMIT_REACHED, "&cMin build height limit reached!");
+        yml.addDefault(Messages.ARENA_FIREBALL_COOLDOWN, "&cPlease wait %bw_cooldown%s to use that again!");
```

Comment:
Your string will look like: Please wait X to do that again. 

x what? I think you forgot to add the word seconds to all lang files

Reply:
> Your string will look like: Please wait X to do that again.
> 
> x what? I think you forgot to add the word seconds to all lang files

![image](https://github.com/tomkeuper/BedWars2023/assets/85287520/a56499dc-a081-4ef5-83ca-3eaea9e78142)